cluster/juju/layers/kubernetes-master/templates/kube-apiserver.defaults

```diff
@@ -5,20 +5,13 @@
 #
 
 # The address on the local server to listen to.
-KUBE_API_ADDRESS="--insecure-bind-address=0.0.0.0"
+KUBE_API_ADDRESS="--insecure-bind-address=127.0.0.1"
```

Comment:
I am +1 to reverting this to localhost access only. This simplifies the user experience, and secures the apiserver much better, its not impervious, and we should probably investigate a firewall rule to reject this port to external connections, but for now, this is a great placeholder fix.